wiki/Beatmap/Difficulty_spread/de.md

```diff
@@ -0,0 +1,20 @@
+---
+tags:
+  - difficulty set
+  - progression
```

Comment:
Note to myself: The tag `progression` is not translated here because the literal translation `Verlauf` implies a time or spatial component, which both does not fit quite well.